### ISSUE TITLE
extensions-0.1.0.1 not compatible with Cabal 3.12 

### ISSUE THREAD

**Issue by soulomoon** [kowainik/extensions#95]
Cabal 3.12 would come with multi home unit support, please consdier release a new version for it.

**Comment by tomjaguarpaw** [comment]
Yes, absolutely. But I don't understand: how does whether `extensions` supports 3.12 or not affect whether you can use 3.12 for other things?

**Comment by soulomoon** [comment]
> Yes, absolutely. But I don't understand: how does whether `extensions` supports 3.12 or not affect whether you can use 3.12 for other things?

Yes, we are trying testing HLS codebase with HLS in cabal 3.12 prerelease https://gitlab.haskell.org/haskell/cabal/-/jobs.
Since cabal 3.12's multiple home component repl impose a constraint. In order to use it, its' package must be able to setup using Cabal >= 3.12 
`extensions` is introduced to HLS as stan's dependencies.

**Comment by tomjaguarpaw** [comment]
I see, that makes sense. Yes, HLS is one of the very few reverse dependencies of `extensions`.

**Comment by tomjaguarpaw** [comment]
Fixed by https://github.com/kowainik/extensions/pull/96 and uploaded to Hackage as https://hackage.haskell.org/package/extensions-0.1.0.2. Please let me know whether that works.

**Comment by soulomoon** [comment]
> Fixed by #96 and uploaded to Hackage as https://hackage.haskell.org/package/extensions-0.1.0.2. Please let me know whether that works.

Thank you for the fast action, we are currently blocking by more https://github.com/haskell/haskell-language-server/issues/4219. To see if it works,  All HLS deps need to support Cabal 3.12. I'll report it when it is done.
